code/go/0chain.net/smartcontract/dbs/event/validator.go

```diff
@@ -24,7 +24,7 @@ type Validator struct {
 	NumDelegates   int           `json:"num_delegates"`
 	ServiceCharge  float64       `json:"service_charge"`
 
-	Reward      int64 `json:"reward"`
+	Rewards     int64 `json:"rewards"`
```

Comment:
doesn't changing the json fails system tests? please ensure that
if it fails, keep the json as it is for now

Reply:
Strangely enough, so far the value seems to be unused by get functions. System tests have also passed.